project/dependencies.scala

```diff
@@ -32,6 +32,10 @@ object Dependencies {
     // sidebar module
     "org.webjars" % "font-awesome" % "4.5.0",
     "org.webjars.bower" % "bootstrap-vertical-tabs" % "1.2.1",
+    // dashboard module
+    "org.webjars.bower" % "angular-dashboard-framework" % "0.11.0",
```

Comment:
They seem to have replied to my issue, let me check if it can be imported right now